playbooks/byo/preflight/action_plugins/verify_openshift_image.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import os
+
+from ansible.plugins.action import ActionBase
+
+
+class ActionModule(ActionBase):
+
+    def run(self, tmp=None, task_vars=None):
+        ''' handler verifying openshift images are available '''
+        if task_vars is None:
+            task_vars = dict()
+
+        result = super(ActionModule, self).run(tmp, task_vars)
+
+        component  = self._task.args.get('component', None)
+        
+        if component is None:
+            result['failed'] = True
+            result['msg'] = "component is required"
+            return result
+
+        docker_image_args = dict(
+                name="openshift3/%s" % component,
```

Comment:
Nothing about this in the project's style guide, but "modern Python" (Python 2.7 and 3.x) uses `string.format` instead of `%`.
https://docs.python.org/2/library/string.html#format-string-syntax